package.json

```diff
@@ -1,33 +1,16 @@
 {
-  "name": "ee-ethereal-village",
+  "name": "ethereal-village",
```

Comment:
The package.json was set up correctly. The name must match the repo name, as well as per our `ee-` naming convention.